openreview/venue_request/venue_request.py

```diff
@@ -1623,6 +1623,19 @@ def setup_request_form(self):
                 'order': 48,
                 'required': False,
                 'hidden': True
+            },
+            'iThenticate_plagiarism_check': {
+                'value-radio': ['Yes', 'No'],
+                'default': 'No',
+                'order': 49,
+                'required': False,
+                'hidden': True
+            },
+            'iThenticate_plagiarism_check_api_key': {
+                'value-regex': '.*',
+                'order': 50,
+                'required': False,
+                'hidden': True
```

Comment:
@akshatchhabra the base url field is missing here.

Reply:
`iThenticate_plagiarism_check_api_base_url`

Reply:
Done